helm/biosdk-service/.gitignore

```diff
@@ -1,2 +1,2 @@
 charts/
-Charts.lock
+Chart.lock
```

Comment:
biosdk-services/deploy
/install.sh  
CHART_VERSION=1.3.0-beta.2-develop
change to 
CHART_VERSION=1.3.0-develop